### ISSUE TITLE
Other medications - to research and write up

### ISSUE THREAD

**Issue by Laeeth** [symmetry-covid/personal-plan#9]
https://docs.google.com/spreadsheets/u/0/d/10rdXbBL4gN72oKTNsWvRkSgxHdJdIj9Ij0pguu9MFwM/htmlview

Albendazole
https://twitter.com/__ice9/status/1285775396852703233?s=19

Colchicine 

Amodiaquine
https://www.wired.com/story/hydroxychloroquine-is-toast-now-say-hi-to-its-dirty-cousin/?utm_source=twitter&utm_medium=social&utm_campaign=onsite-share&utm_brand=wired&utm_social-type=earned

Famotidine

Nitazoxanide

Favipiravir

Ciclesonide 

Ivermectin vet
https://twitter.com/jaysanchezdorta/status/1280312098246086657?s=19 

Injectable Artesunate 

Ecosprin

Ceftriaxone

https://www.newindianexpress.com/states/karnataka/2020/sep/19/hcq-no-longer-a-magic-pill-doctors-move-to-new-drugs-2198937.amp?__twitter_impression=true  

Arbidol / umifenovir
https://rupharma.com/
https://twitter.com/__ice9/status/1316112583754158088?s=19 

Russia has widespread availability of Arbidol (umifenovir) and Grippferon (interferon alpha-2b), both of which have been shown in Chinese studies to have some efficacy against COVID-19 (nebulized interferon though).

Bromhexine and tilorone also look very likely to be useful.

Regarding the thymus, a section on short peptide bioregulators would be useful.

They generally just increase expression of particular genes, usually transiently via binding to a promoter.  For instance, epitalon is a telomerase activator.

Other Russian stuff:

- tilorone is a more intensive interferon inducer

- various interferon products:
https://t.co/t8LbljSfF1

There is another interferon brand whose name I cannot remember.  It is like Grippferon but about 2.5x more potent.  Worth looking for

There are no human clinical studies for it against COVID-19.  The usual dose is 125mg.  Half life is 48 hours.

The effect in stimulating interferon response is potent and systemic.  Such a strong interferon inducer should not be used after a few days of symptoms have elapsed.

In other words, it is the kind of thing that might make sense as a 'hit-and-run' interferon ramp-up strategy at the beginning of a symptomatic viral infection, not as much for continued use throughout its course.

Other, milder options include e.g. elderberry syrup and umifenovir

https://twitter.com/__ice9/status/1336879200993730563?s=19

Correct.  The fastest source of ambroxol is Mexico.  Mexican pharmacies also carry other useful/relevant items.  I tend to order mostly from India as selection and price are usually a bit better, but have bought from: India, Mexico, Turkey, Japan, Russia, Latvia, Thailand, etc.
